USAGE.md

```diff
@@ -32,10 +32,10 @@ Git directories are searched for the latest commit hash. Searching for git commi
 
 ### Ignored files
 
-By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, `.gitignore` files are parsed recursively starting from the target directory,
-otherwise they are parsed from the root of the git repository as typical.
+By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, any `.gitignore` files in the target directory and all subdirectories will be parsed,
```

Comment:
Can you update this to reflect our latest discussion in https://github.com/google/osv-scanner/pull/206#discussion_r1103972400 ?